### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by zackforbing**
changed the title of the existing cart test to use 'visitor' instead of 'user' for clarity.

### CODE REVIEW

File: spec/features/visitor_can_view_categories_page_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'Visitor can view category pages', type: :feature do
```
Comment (karinamzalez) [inline spec/features/visitor_can_view_categories_page_spec.rb]: Again, for the sake of consistency, we should talk about choosing one phrase together-- I used "guest user" in my tests.

Comment (concach) [inline spec/features/visitor_can_view_categories_page_spec.rb]: "Visitor" is what we used in our Waffle cards - I vote we keep that consistent

**Comment by concach** [comment]
I approve a merge. Note that we use "visitor" in waffle cards, so these are consistent ... when in doubt, I think we should stick with outlines or code we've already created

**Comment by kbs5280** [comment]
Looks good. Merging.

**Pull request merged**